### PR TITLE
Add column 'last_login' to user model

### PR DESCRIPTION
**Pull request by thyseus**
The time of the last login is saved in the database upon successful login.

### CODE REVIEW

File: migrations/m160929_103127_add_last_login_to_user_table.php [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?php
+
+use yii\db\Migration;
+
+class m160929_103127_add_last_login_to_user_table extends Migration
+{
+  public function up()
+  {
+    $this->addColumn('user', 'last_login', $this->integer());
```
Comment (dmeroff) [inline migrations/m160929_103127_add_last_login_to_user_table.php]: I think it's better to name it like `last_login_at`.

File: views/admin/index.php [hunk 2]
```diff
@@ -53,13 +53,24 @@
...
+                else
                     return date('Y-m-d G:i:s', $model->created_at);
-                }
             },
         ],
+
+        [
+          'attribute' => 'last_login',
+          'value' => function ($model) {
+            if(!$model->last_login || $model->last_login == 0)
```
Comment (dmeroff) [inline views/admin/index.php]: This code block must be formatted with PSR-2 codestyle.

File: views/admin/index.php [hunk 3]
```diff
@@ -53,13 +53,24 @@
         [
             'attribute' => 'created_at',
             'value' => function ($model) {
-                if (extension_loaded('intl')) {
```
Comment (dmeroff) [inline views/admin/index.php]: This code block must be formatted with PSR-2 codestyle.

**Comment by thyseus** [comment]
attribute is renamed and master is merged into branch

**Comment by thyseus** [comment]
Rookie mistake. Forgot to do "git add" after renaming the file. Good that you have an sharp eye :)

**Comment by dmeroff** [comment]
Thank you!

**Pull request merged**